platform/viewer/src/components/Header/Header.js

```diff
@@ -123,21 +128,11 @@ class Header extends Component {
               {t('INVESTIGATIONAL USE ONLY')}
             </span>
             <Dropdown title={t('Options')} list={this.options} align="right" />
-
-            {/* TODO: We need a Modal service */}
-            <AboutModal
-              {...this.state}
-              onCancel={() =>
-                this.setState({
-                  isOpen: false,
-                })
-              }
-            />
           </div>
         </div>
       </>
     );
   }
 }
 
-export default withTranslation('Header')(withRouter(Header));
+export default withTranslation('Header')(withRouter(withModal(Header)));
```

Comment:
Worth making this a functional component?

Reply:
I think it's worth it, but I see conflicts with hotkeys branch, I can refactor it in the hotkeys branch after we merge this one. What do you think?